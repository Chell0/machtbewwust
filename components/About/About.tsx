export default function About() {
    return (
        <div className="bg-white p-5 rounded-lg shadow-lg mx-auto w-full max-w-[1017px]">
            <p className="text-gray-600 text-md">
                Willkommen auf dem Blog MachtBewusst-Hamburg! Hier dreht sich alles um machtkritische und dekoloniale Perspektiven der Hamburger Zivilgesellschaft. Wir möchten das Engagement und die Projekte im Dekolonisierungsprozess in Hamburg hervorheben und Euch mit interessanten Beiträgen, Fachwissen und Erfahrungen inspirieren. Egal ob, Ihr bereits aktiv oder einfach neugierig seid, hier findet Ihr eine Datenbank mit Akteuren:innen, einen Veranstaltungskalender und spannende Materialien zur Thematik. Schaut vorbei, informiert Euch und lasst Euch von neuen Gedanken anregen.
            </p>
        </div>
    );
}
