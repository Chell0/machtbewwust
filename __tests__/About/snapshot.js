import { render } from "@testing-library/react";
import About from "@/components/About/About";

// Snapshot test case
it("renders aboutpage unchanged", () => {
	// Render the About component and get a fragment of the rendered output
	const { container } = render(<About />);

	// Assert that the rendered output matches the snapshot
	expect(container).toMatchSnapshot();
});
