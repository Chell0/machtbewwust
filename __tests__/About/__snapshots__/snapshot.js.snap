// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders aboutpage unchanged 1`] = `
<div>
  <div
    class="bg-white p-5 rounded-lg shadow-lg mx-auto  w-full sm:w-[1016.95px]"
  >
    <p
      class="text-left text-md text-gray-600"
    >
      Willkommen auf dem Blog MachtBewusst-Hamburg! Hier dreht sich alles um machtkritische und dekoloniale Perspektiven der Hamburger Zivilgesellschaft. Wir möchten das Engagement und die Projekte im Dekolonisierungsprozess in Hamburg hervorheben und euch mit interessanten Beiträgen, Fachwissen und Erfahrungen inspirieren. Egal ob ihr bereits aktiv oder einfach neugierig seid, hier findet ihr eine Datenbank mit Akteuren:innen, einen Veranstaltungskalender und spannende Materialien zur Thematik. Schaut vorbei, informiert euch und lasst euch von neuen Gedanken anregen.
    </p>
  </div>
</div>
`;
